src/test/java/org/folio/rest/impl/PatronResourceImplTest.java

```diff
@@ -843,6 +855,33 @@ public final void testPostPatronAccountByIdItemByItemIdRequests(String itemId, S
     logger.info("Test done");
   }
 
+  @Test
+  public final void testPostPatronAccountByIdItemByItemIdRequestsPatronComments() {
+    logger.info("Testing creating a hold with patron comments on an item for the specified user");
+
+    final var expectedPatronComment = "I need this book. Item level comment.";
+
+    final var holdResponse = given()
+        .log().all()
+        .headers(new Headers(tenantHeader, urlHeader, contentTypeHeader,
+                 new Header("x-okapi-patron-comment", expectedPatronComment)))
```

Comment:
Why do we need this header?

Reply:
This just to provide expected patron comment to server's stub.  I used this approach to pass patron comment to response mock handler to add it dynamically to response.

Reply:
bq. Would it be better to just response with prepared payload contained patron comment instead?

Why is a prepared response needed? What do the other tests do that check the interaction with the circulation interface?

Reply:
I saw there was some tests passes some kind of "bad-data" via headers to trigger different checking and then on server's stub just processing it. I thought this could be used instead of extra file creation. Ok, I'll use separate payload response for this instead. Thanks.